src/puppetlabs/puppetdb/http/handlers.clj

```diff
@@ -252,12 +252,19 @@
 (pls/defn-validated packages-routes :- bidi-schema/RoutePair
   [version :- s/Keyword]
   (extract-query
+   (cmdi/routes
+    (cmdi/ANY "" []
+              (create-query-handler version "packages")))))
+
+(pls/defn-validated package-inventory-routes :- bidi-schema/RoutePair
+  [version :- s/Keyword]
+  (extract-query
     (cmdi/routes
       (cmdi/ANY "" []
-                (create-query-handler version "packages"
+                (create-query-handler version "package_inventory"
```

Comment:
what's the significance of the underscore here, relative to the dash in server.clj?